strings.txt

```diff
@@ -12847,7 +12864,7 @@ BROWSE_MUSIC
 	EN	Music Library
 	ES	Biblioteca de música
 	FI	Musiikkikirjasto
-	FR	Bibliothèque musicale
+	FR	Bibliothèque multimédia
```

Comment:
Are you sure about this change? It's the menu item shown on the players etc., where you can only access music files, but none of the other media types (movies, photos).